docs/Deployment/BackupRestore.md

```diff
@@ -0,0 +1,167 @@
+# Backup & Restore
+
+This guide gets you up to speed with the [Docker deployment](../Docker/)'s Backup & Restore process.
+You will learn about what to keep in mind when doing a backup and how to restore a backup.
+
+[[toc]]
+
+## Remarks and Considerations
+
+The original backups strategy in BTCPay Server still exists and can be found [here](https://docs.btcpayserver.org/Docker/#how-can-i-back-up-my-btcpay-server).
```

Comment:
this page need to have a BIG warning pointing out to this new process. (or maybe just delete the old process?)

Reply:
Updated that part in btcpayserver/btcpayserver-docker#655.